### ISSUE TITLE
Create new folder doesnt work when run from a Win32 app -> .NET

### ISSUE THREAD

**Issue by hdsoftware** [NLog/NLog#2083]
**Type** - Bug
**NLog version**: (e.g. 4.3.11)
**Platform**: .Net 4.5

Hi..
We have a strange problem.  If I run my application and the folder receiving the logs doesnt exist, then my application get stucked.  At least it looks like it does because its not responding.  My guess is that thers some kind of timeout and my app posts around 100 INFO's at startup.  Anyway, I have let the app go for 30 minutes and still nothing.   So, i terminate the process, adds the directory and restarts.  Everything is ok.

Ok.  Looking in the docs and thers a tag that enables autocreate, but the doc says its default Enabled.  I still tryed to enable it, just to check..

Now, thers a big difference between this application and a normal .net application.  I'm using Clarion for Windows to create the main application.  It creates standard Win32 binary like C.  Its allso compatible with exported functions in C and vice versa.  I'm using a NuGet component to export the NLOG API to my program.

In short:
program.EXE is NOT .NET
Logging component that use NLOG is .NET
My method LOG(string message) is exported from my .NET project and called from the Win32 program

Any tips here?  Or is this a bug ?

**Comment by 304NotModified** [comment]
Could you create a trace internal log? https://github.com/NLog/NLog/wiki/Internal-logging

**Comment by hdsoftware** [comment]
Il give that a try tuesday..

Sendt fra min iPad

Den 28. apr. 2017 kl. 23.00 skrev Julian Verdurmen <notifications@github.com<mailto:notifications@github.com>>:


Could you create a trace internal log? https://github.com/NLog/NLog/wiki/Internal-logging

**Comment by 304NotModified** [comment]
Closing this due to inactivity. Please let us know if this still an issue and please provide the requested info.
